library/php-zendserver

```diff
@@ -1,6 +1,6 @@
-Maintainers: David Lowes <david.l@zend.com> (@davidl-zend)
+Maintainers: Andrus Viik <andrus.viik@roguewave.com> (@andrus7k)
```

Comment:
I'll use your commit rights on https://github.com/zendtech/php-zendserver-docker as validation to make the maintainer change. If @davidl-zend wants to revert or change this, then please let us know.